connectors/byoc.py

```diff
@@ -55,19 +59,6 @@ class JobTriggerMethod(Enum):
     UNSET = None
 
 
-CUSTOM_READ_ONLY_FIELDS = (
```

Comment:
It was removed because this logic is now a part of Connector class and properties/methods that it provides to mutate the object state.